src/utils/object.js

```diff
@@ -37,4 +37,17 @@ export const pick = (obj, ...keys) => {
     (searchKeys => searchKeys.indexOf(key) > -1)(flattendKeys));
 };
 
+export const omit = (obj, ...keys) => {
```

Comment:
not used

Reply:
Maybe we can refactor in the next iteration. Currently react grid uses it.